ReaScript.cpp

```diff
@@ -301,7 +301,7 @@ APIdef g_apidefs[] =
 	{ APIFUNC(NF_UpdateSWSMarkerRegionSubWindow), "void", "", "", "Redraw the Notes window (call if you've changed a subtitle via <a href=\"#NF_SetSWSMarkerRegionSub\">NF_SetSWSMarkerRegionSub</a> which is currently displayed in the Notes window and you want to appear the new subtitle immediately.)", },
 
 	{ APIFUNC(NF_TakeFX_GetFXModuleName), "bool", "MediaItem*,int,char*,int", "item,fx,nameOut,nameOut_sz", "Deprecated, see TakeFX_GetNamedConfigParm/'fx_ident' (v6.37+). See BR_TrackFX_GetFXModuleName. fx: counted consecutively across all takes (zero-based).", },
-	{ APIFUNC(NF_Win32_GetSystemMetrics), "int", "int", "nIndex", "Equivalent to win32 API GetSystemMetrics().", },
+	{ APIFUNC(NF_Win32_GetSystemMetrics), "int", "int", "nIndex", "Equivalent to win32 API GetSystemMetrics(). Note: Only a small subset of the parameters is supported in SWELL and therefore when using this function on MacOS or Linux, see <a href=\"https://forum.cockos.com/showthread.php?p=2559455#post2559455\">forum post</a>.", },
```

Comment:
I like the idea but though it's factical correct info I think it's not really a future-proof statement (in case new additions are made to SWELL's GetSystemMetrics in the future ReaScripter's wouldn't be aware of it unless we update the decription regularly which I don't quite see to happen. :D)
So maybe in addition with link to Justin's/Cockos WDL repo?

s.th. like
`Also see the <a href=\"https://forum.cockos.com/showthread.php?p=2559455#post2559455\">the Cockos WDL repo</a> for potential additions.`
?

But either way is fine for me.

Reply:
> edit: Hm...line numbers can change so maybe [link to main branch instead of fixed commit]

👍

 However the macOS and Linux implementations (swell-wnd-generic.cpp) aren't guaranteed to always support the same parameters.

How about linking to the folder instead?

```html
Check the <a href="https://github.com/justinfrankel/WDL/blob/main/WDL/swell">SWELL source code</a> for up-to-date support information (swell-wnd.mm, swell-wnd-generic.cpp).
```